src/components/PaperWallet.tsx

```diff
@@ -197,7 +197,7 @@ interface Props {
   isHidden?: boolean;
 }
 
-export default class PaperWallet extends React.Component<Props, {}> {
+export default class PaperWallet extends React.Component<Props, TObject> {
```

Comment:
If the component doesn't have state, I'd just use `null` here.

```suggestion
export default class PaperWallet extends React.Component<Props, null> {
```

Reply:
Done